MessengerOnFileForLinux/UserService/test/SignOutTestCore.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <gtest/gtest.h>
+
+#include "SignOut.hpp"
+
+class SignOutTestFixture : public ::testing::Test
+{
+public:
+    SignOutTestFixture() {}
+    void SetUp() {}
+    void TearDown() {}
+    ~SignOutTestFixture() {}
+
+    SignOut signOut;
```

Comment:
Zastanawiam sie czy nie nazywać takich memberów sut albo cos podobnego, żeby było wiadomo co jest testowane. Tu to róznicy nie robi, ale np. w SignIn już mamy troche więcej memberów w fixturce

Reply:
nie rozumiem